pkg/operation/seed/scheduler/types.go

```diff
@@ -14,6 +14,31 @@
 
 package scheduler
 
-// GardenerShootControlPlaneSchedulerName is the name of the scheduler used for
-// shoot control plane pods.
-const GardenerShootControlPlaneSchedulerName = "gardener-shoot-controlplane-scheduler"
+import (
+	"github.com/Masterminds/semver"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+)
+
+const (
+	// GardenerShootControlPlaneSchedulerName is the name of the scheduler used for
+	// shoot control plane pods.
+	GardenerShootControlPlaneSchedulerName = "gardener-shoot-controlplane-scheduler"
+	// GardenerShootControlPlaneSchedulerWebhookPath is the path of the webhook server
+	// that sets the "gardener-shoot-controlplane-scheduler" schedulerName for Pods.
+	GardenerShootControlPlaneSchedulerWebhookPath = "/webhooks/default-pod-scheduler-name/" + GardenerShootControlPlaneSchedulerName
```

Comment:
Can you have a constant with the path somwhere where in `pkg/seedadmission` and use it here and [here](https://github.com/gardener/gardener/blob/f8f07b321caa6c526d367947eaddf47ec5744c5f/cmd/gardener-seed-admission-controller/app/gardener_seed_admission_controller.go#L155)?

Reply:
I would rather keep the constant here as its about the scheduler.

Reply:
Hm, but the webhook path is defined by the webhook server which is in `cmd/gardener-seed-admission-controller/app`, so I the constant definitely belongs there (or rather in `pkg/seedadmission`). The gardener-kube-scheduler just uses it, but it does not define it.